src/test/java/de/uhd/ifi/se/decision/management/jira/view/treeviewer/TestData.java

```diff
@@ -33,6 +33,32 @@ public void testConstructorWithElementAndLink() {
 		assertEquals("tv1", data.getId());
 	}
 
+	@Test
+	public void testConstructorWithDescNull() {
+		KnowledgeElement element2 = new KnowledgeElementImpl();
+		element2.setId(1);
+		element2.setSummary("Test-12");
+		element2.setType("Other");
+	}
+
+	@Test
+	public void testConstructorWithDescBlank() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5229259)